packages/grafana-ui/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { css } from 'emotion';
+import { stylesFactory } from '../../themes';
+import { Button, ButtonVariant } from '../Forms/Button';
+
+interface Props {
+  visible?: boolean;
+  currentPage: number;
+  numberOfPages: number;
+  onNavigate: (toPage: number) => void;
+}
+
+const getStyles = stylesFactory(() => {
+  return {
+    container: css`
+      float: right;
+    `,
+    item: css`
+      display: inline-block;
+      padding-left: 10px;
+      margin-bottom: 5px;
+    `,
+  };
+});
+
+export const Pagination: React.FC<Props> = ({ visible = true, currentPage, numberOfPages, onNavigate }) => {
+  if (visible === false) {
+    return null;
+  }
```

Comment:
Hmm, maybe instead of passing `visible` prob, we should rather just not render the component..?:) WDYT?